### DISCUSSION TITLE
Received newsletter for yetis: YetiForce 7.0 is available!

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#17425] [General]
It is pretty ambiguous, as I am asked to Order Audit (I guess fee follows), no repository shown, no demo has been released. 
Is it Open Source release? What is the trick?

**Comment by FranziskaSom** [comment]
we can't find a demo version either ... link goes to 6.5

**Comment by paula-w** [comment]
Hi guys, just FYI - https://github.com/YetiForceCompany/YetiForceCRM/discussions/17427 
you can find all the relevant links there 👍
